build.gradle.kts

```diff
@@ -0,0 +1,47 @@
+import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
+
+plugins {
+    kotlin("multiplatform") version "1.6.10"
+    id("maven-publish")
+}
+
+group = "pl.allegro.mobile"
+version = "1.0.0"
```

Comment:
nope, nope nope ;) maybe we could use [axion-release](https://github.com/allegro/axion-release-plugin)?

Reply:
Also I think for now, before we're ready to ship this in production -  we'll need to use 0.X.Y versions. IMO 1.0.0 should be released only when initial, production-ready version is ready to ship.

Reply:
Done, with axion the very first version will be 0.1.0